tests/roles/manila_adoption/tasks/main.yaml

```diff
@@ -1,19 +1,63 @@
+- name: Check the required input when manila_backend is NFS
+  when: manila_backend == "cephnfs"
+  block:
+    - name: Fail if no CephNFS VIP has been provided as input
+      when:
+        - manila_nfs_vip is undefined
+      ansible.builtin.fail:
+        msg: "No CephNFS VIP has been provided"
+    - name: Fail if no target nodes have been provided as input
+      when:
+        - manila_nfs_nodes is undefined
+      ansible.builtin.fail:
+        msg: "No Ceph Target nodes have been provided"
+
+    - name: set shell vars
+      no_log: "{{ use_no_log }}"
+      ansible.builtin.set_fact:
+        controller_ssh: |
+          CONTROLLER1_SSH="{{ controller1_ssh }}"
+
+    - name: Get ceph-nfs IP Address
+      become: true
+      ansible.builtin.shell: |
+        {{ shell_header }}
+        {{ controller_ssh }}
+        ${CONTROLLER1_SSH} sudo pcs constraint config | awk '/^  ip-.*ceph-nfs/{split($1,out,"-"); print out[2]}'
```

Comment:
@gouthampacha I'm not sure we have a better (less error prone) way to gather this IP address. I can experiment a bit but I'm open to suggestions.
An idea is to grep it from `/etc/ganesha/ganesha.conf`: not sure we have other challenges there or is less complicated.

Reply:
Looks better now